File: use-cases/vendor-direct-fulfillment/code/csharp/src/sp-api-csharp-app/lambda/utils/AppCredentials.cs
using System;
using Newtonsoft.Json;

namespace spApiCsharpApp
{
    public class AppCredentials
    {
        [JsonProperty("AppClientId")]
        public String clientId;

        [JsonProperty("AppClientSecret")]
        public String clientSecret;

        [JsonProperty("AppRefreshToken")]
        public String refreshToken;

    }
}

File: use-cases/notifications/code/java/src/main/java/lambda/common/AppCredentials.java
package lambda.common;

import com.google.gson.annotations.SerializedName;
import lombok.Getter;

@Getter
public class AppCredentials {

    @SerializedName("AppClientId")
    private String clientId;

    @SerializedName("AppClientSecret")
    private String clientSecret;

    public AppCredentials(String clientId, String clientSecret) {
        this.clientId = clientId;
        this.clientSecret = clientSecret;
    }
}

